lib/Elastica/Suggest/AbstractSuggest.php

```diff
@@ -31,15 +37,29 @@ public function __construct($name, $field)
     }
 
     /**
-     * Suggest text must be set either globally or per suggestion.
+     * Suggest prefix must be set either globally or per suggestion.
+     *
+     * @param string $prefix
+     *
+     * @return $this
+     */
+    public function setPrefix($prefix)
+    {
+        $this->_prefix = $prefix;
+
+        return $this;
+    }
+
+    /**
+     * Suggest regex must be set either globally or per suggestion.
      *
-     * @param string $text
+     * @param string $regex
      *
      * @return $this
      */
-    public function setText($text)
```

Comment:
If I look at the 5.0 examples, `text` shows up quite frequently: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters.html